### PR TITLE
#MYSQL IMAGE CHANGE

### PR DESCRIPTION
**Pull request by brtbach**

### CODE REVIEW

**Comment by brtbach** [comment]
switch to linuxserver image (pgid&puid jail)
changed valume according to linuxsever docker image
fixed version to 10.5.11-r0 (newer versions cause issues with nextcloud)
disabled auto update

File: apps/addons/mariadb.yml [hunk 1]
```diff
@@ -9,8 +9,8 @@ services:
       - 'PUID=${ID}'
       - 'TZ=${TZ}'
       - 'UMASK=${UMASK}'
-      - 'MYSQL_ROOT_PASSWORD=${MARIADB_ROOT_PASSWORD}'
-    image: 'mariadb:latest'
+      - 'MYSQL_ROOT_PASSWORD=password'
```
Comment (doob187) [inline apps/addons/mariadb.yml]: Change  this to environment back

File: apps/addons/mariadb.yml [hunk 2]
```diff
@@ -9,8 +9,8 @@ services:
...
+    image: 'linuxserver/mariadb:amd64-alpine-version-10.5.11-r0'
```
Comment (doob187) [inline apps/addons/mariadb.yml]: Add this environment.
Not all runs amd versions 
Some runs it on arm

**Review by doob187 — CHANGES_REQUESTED** [review]
See Infos

**Comment by drag0n141** [comment]
@doob187 it is all fixed :)

**Review by doob187 — COMMENTED** [review]
#patch

**Pull request merged**